src/libs/cropOrRotateImage/index.ts

```diff
@@ -1,127 +1,19 @@
-    // elements. Transformations such as rotation have to be applied before drawing
-    const result = document.createElement('canvas');
-    result.width = width;
-    result.height = height;
-
-    const context = result.getContext('2d');
-    if (context) {
-        // In order to rotate image along its center we have to apply next transformation
-        context.translate(result.width / 2, result.height / 2);
-
-        const radians = (degrees * Math.PI) / 180;
-        context.rotate(radians);
-
-        context.drawImage(canvas, -canvas.width / 2, -canvas.height / 2, canvas.width, canvas.height);
-    }
-    return result;
-}
-
-/**
- * Creates new cropped canvas and returns it
- */
-function cropCanvas(canvas: HTMLCanvasElement, options: CropOptions) {
-    let {originX = 0, originY = 0, width = 0, height = 0} = options;
-    const clamp = (value: number, max: number) => Math.max(0, Math.min(max, value));
-
-    width = clamp(width, canvas.width);
-    height = clamp(height, canvas.height);
-    originX = clamp(originX, canvas.width);
-    originY = clamp(originY, canvas.height);
-
-    width = Math.min(originX + width, canvas.width) - originX;
-    height = Math.min(originY + height, canvas.height) - originY;
-
-    const result = document.createElement('canvas');
-    result.width = width;
-    result.height = height;
-
-    const context = result.getContext('2d');
-    context?.drawImage(canvas, originX, originY, width, height, 0, 0, width, height);
-
-    return result;
-}
-
-function convertCanvasToFile(canvas: HTMLCanvasElement, options: CropOrRotateImageOptions): Promise<File> {
-    return new Promise((resolve) => {
-        canvas.toBlob((blob) => {
-            if (!blob) {
-                return;
-            }
-            const file = new File([blob], options.name || 'fileName.jpeg', {type: options.type || 'image/jpeg'});
-            file.uri = URL.createObjectURL(file);
-            resolve(file);
-        });
-    });
-}
-
-/**
- * Loads image from specified url
- */
-function loadImageAsync(uri: string): Promise<HTMLCanvasElement> {
-    return new Promise((resolve, reject) => {
-        const imageSource = new Image();
-        imageSource.crossOrigin = 'anonymous';
-        const canvas = document.createElement('canvas');
-        imageSource.onload = () => {
-            canvas.width = imageSource.naturalWidth;
-            canvas.height = imageSource.naturalHeight;
-
-            const context = canvas.getContext('2d');
-            context?.drawImage(imageSource, 0, 0, imageSource.naturalWidth, imageSource.naturalHeight);
-
-            resolve(canvas);
-        };
-        imageSource.onerror = () => reject(canvas);
-        imageSource.src = uri;
-    });
-}
-
-/**
- * Crops and rotates the image on web
- */
+import {manipulateAsync} from 'expo-image-manipulator';
+import getSaveFormat from 'src/libs/cropOrRotateImage/getSaveFormat';
```

Comment:
```suggestion
import getSaveFormat from './getSaveFormat';
```

This path is incorrect and breaking the build.